<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_userName">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_password_reenter">
        <property name="text">
         <string>密码确认：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_picture">
        <property name="text">
         <string>头像选取</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="MyLineEdit" name="lineEdit_userName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLineEdit" name="lineEdit_password">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password_reenter"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_picture"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_picture">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_register_cpt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>完成注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pictureSelected">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选取的头像</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">UI/MyLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
